packages/framework/src/Model/Mail/MailTemplateBuilder.php

```diff
@@ -75,6 +75,21 @@ protected function getFooterIcons(int $domainId): string
      */
     public function getMailTemplateWithContent(int $domainId, ?string $content = ''): string
     {
+        $footerIconsHtml = $this->getFooterIcons($domainId);
+        $footerIconsTable = '';
+
+        if (!empty($footerIconsHtml)) {
```

Comment:
I think the empty is not allowed by standards. Please use check for a more specific value